src/TextField/TextFieldLabel.js

```diff
@@ -4,15 +4,14 @@ import React, { PropTypes } from 'react';
 import { createStyleSheet } from 'jss-theme-reactor';
 import classNames from 'classnames';
 import { easing } from '../styles/transitions';
+import LabelBase from '../internal/LabelBase';
```

Comment:
Split `TextFieldLabel` basics into `LabelBase` for use in other form situations such as a checkbox.